hs_core/templates/pages/genericresource.html

```diff
@@ -1587,9 +1587,11 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             <div class="tab-pane" id="relevantTools">
                             {% if not metadata_form %}
                                 {% for tool in relevant_tools %}
-                                    <h4>{{ tool.title }}</h4>
-                                    <a href="{{ tool.url }}" target="_blank"><button class="btn btn-primary">Launch App</button></a>
-                                    <hr>
+                                    <div style="display:inline-block; padding-left:20px; text-align:center;">
+                                        <h4>{{ tool.title }}</h4>
+                                        <a href="{{ tool.url }}" target="_blank"><img src="{{ tool.icon }}" style="width:100px; height:100px;"
```

Comment:
Same here, move style into css if there is not one already defined in the css file.